src/db/parity_db.rs

```diff
@@ -140,16 +190,36 @@ impl Blockstore for ParityDb {
         D: AsRef<[u8]>,
         I: IntoIterator<Item = (Cid, D)>,
     {
-        let values = blocks
+        let values = blocks.into_iter().map(|(k, v)| {
+            let column = Self::choose_column(&k);
+            match column {
+                DbColumn::GraphDagCborBlake2b256 | DbColumn::Settings | DbColumn::GraphFull => {
```

Comment:
Is this defensive coding in case new columns are added? Can't we rely on `Self::choose_column` always picking the right column?

Reply:
I believe it's kind of a leftover. I'll change it.